dotnet/samples/Demos/VectorStoreRAG/README.md

```diff
@@ -45,6 +48,7 @@ For Azure OpenAI, you need to add the following secrets:
 
 ```cli
 dotnet user-secrets set "AIServices:AzureOpenAI:Endpoint" "https://<yourservice>.openai.azure.com"
+dotnet user-secrets set "AIServices:AzureOpenAI:ChatDeploymentName" "gpt-4"
```

Comment:
Deployment names can be anything, we typically use the model id but customers often use different names.

To void confusion consider:
`dotnet user-secrets set "AIServices:AzureOpenAI:ChatDeploymentName" "<your deployment name>"`

Reply:
Sure, I was in between wanting to give people an example and it being confusing having an actual value in there.  Happy to add the placeholder.